src/components/EditInputGroup.vue

```diff
@@ -58,6 +58,23 @@
 </template>
 
 <script>
+const findCommonPrefix = lines => {
+    // Find the substring common to the array of strings
+    // Inspired from https://stackoverflow.com/questions/68702774/longest-common-prefix-in-javascript
+
+    // Check border cases size 1 array and empty first word)
+    if (!lines[0] || lines.length ===  1) return lines[0] || ""
+
+    // While all lines have the same character at position i, increment i
+    let i = 0
+    while (lines[0][i] && lines.every(w => w[i] === lines[0][i])) {
+        i += 1
+    }
```

Comment:
Here is some problem with the code. This is a code smell of eslint [no-loop-func](https://eslint.org/docs/rules/no-loop-func). I do not get the details exactly. Maybe you can check this a bit more in detail.